lib/matplotlib/text.py

```diff
@@ -240,7 +240,8 @@ def update(self, kwargs):
         """
         bbox = kwargs.pop('bbox', None)
         super(Text, self).update(kwargs)
-        self.set_bbox(bbox)  # depends on font properties
+        if bbox:
```

Comment:
If that is the case, we have a whole bunch of other places in the code have the same issue.